advocacy_docs/supported-open-source/barman/single-server-streaming/step01-db-setup.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: "Backup and Recovery: Single-Server Streaming - Configuring PostgreSQL"
+description: "Configuring a PostgreSQL server for Barman streaming replication"
+navTitle: "Database Server Configuration"
+product: barman
+platform: ubuntu
+tags:
+    - ubuntu
+    - barman
+    - postgresql
+    - streaming-replication
+    - live-demo
+katacodaPanel:
+    scenario: ubuntu:2004
+    initializeCommand: wget -q https://raw.githubusercontent.com/josh-heyer/barmantutorial/main/step01-db-setup/docker-compose.yaml ; docker-compose up -d ; clear ; docker exec -it -u postgres pg /bin/bash
+    codelanguages: shell, sql, ini
+iconName: tutorial
+---
+
+This is an interactive demonstration. You can follow along right in your browser, using Katacoda, by clicking the button below:
+
+<KatacodaPanel />
+
+!!! Note
+    The interactive demo loads a Docker Compose application with two container images representing a
+    PostgreSQL 12 server with [the Pagila database](https://github.com/devrimgunduz/pagila) loaded (named `pg`)
```

Comment:
I noticed in some of the examples it seems like we're using PG 13?

Reply:
yep, good catch. @gbartolini just pointed this out to me as well!